doc/release-notes-rbf-stuff.md

```diff
@@ -0,0 +1,9 @@
+# RPC
```

Comment:
In fa16051eac9a4bc1a7e0234b65d615e655ff7cf0 "rpc: Allow fullrbf fee bump"

Please name the file following the expected release notes fragment naming convention.